### PR TITLE
Connect "About Me" update with the Datastore. Add links to the names from Chat

### PR DESCRIPTION
**Pull request by tharminhtet**
- Edited Chat.jsp so that the names in the Conversation are linked to their profile pages.

- Make changes in ProfileServlet and datastore related files to update the User entity with a new updated User entity (i.e. with updated About Me)

### CODE REVIEW

File: src/test/java/codeu/controller/ProfileServletTest.java [hunk 1]
```diff
@@ -70,4 +70,9 @@ public void testDoGet() throws IOException, ServletException{
     Mockito.verify(mockRequest).setAttribute("messages", fakeMessageList);
     Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
   }
+
+  @Test
+  public void testDoPost() throws IOException, ServletException{
+
+  }
 }
```
Comment (sherryx8) [inline src/test/java/codeu/controller/ProfileServletTest.java]: don't forget to finish this?

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 2]
```diff
@@ -121,4 +121,18 @@ public boolean isUserRegistered(String username) {
...
+      if (user.getName().equals(username)) {
+        users.remove(user);
+        users.add(newUser);
+        persistentStorageAgent.update(aboutMe, username);
+        return;
+      }
+    }
+
+
+  }
```
Comment (sherryx8) [inline src/main/java/codeu/model/store/basic/UserStore.java:137]: is there a way to do this without removing the user?

**Review by sherryx8 — APPROVED** [review]
This is really impressive! really good work! 👍

File: src/main/java/codeu/model/store/basic/UserStore.java (hunk 2 above)
Comment (tharminhtet) [reply src/main/java/codeu/model/store/basic/UserStore.java:137]: I've been thinking about it for a while, but haven't found a solution yet. You have an idea?

**Pull request merged**